exercises/concept/elyses-enchantments/.meta/config.json

```diff
@@ -7,5 +7,5 @@
     "test": ["enchantments.spec.js"],
     "exemplar": [".meta/exemplar.js"]
   },
-  "forked_from": ["go/basic-slices"]
+  "forked_from": ["go/card-tricks"]
```

Comment:
This one is still pending (https://github.com/exercism/go/pull/1464)

Reply:
Should probably wait then until that's merged.

Reply:
Was merged :)